modules/apps/transfer/types/forwarding_info_test.go

```diff
@@ -0,0 +1,151 @@
+package types_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/cosmos/ibc-go/v8/modules/apps/transfer/types"
+	host "github.com/cosmos/ibc-go/v8/modules/core/24-host"
+	ibctesting "github.com/cosmos/ibc-go/v8/testing"
+)
+
+var validHop = &types.Hop{
+	PortId:    "transfer",
+	ChannelId: "channel-0",
+}
+
+func TestForwardingInfo_Validate(t *testing.T) {
+	tests := []struct {
+		name           string
+		forwardingInfo *types.ForwardingInfo
+		expError       error
+	}{
+		{
+			"valid msg no hops",
```

Comment:
Instead of `msg` use `forwarding info` in the test name?

```suggestion
			"valid forwarding info with no hops",
```